src/js/firstDiscoveryServer.js

```diff
@@ -21,10 +21,15 @@ var fdSrcDir = path.resolve(__dirname, "../../node_modules/first-discovery/src")
 
 fluid.defaults("gpii.firstDiscovery.server", {
     gradeNames: ["gpii.express"],
+    port: 8080,
```

Comment:
This default doesn't agree with the one in vars.yml and there still seems to be no attempt to keep these two port numbers in step. Recall that the mission is to provide a "one stop shop" - a single place in a single configuration file that the user needs to change in order to get a fully functioning server inside the container, running on a port of their choosing